portfolio-project-matching-app/components/Button.js

```diff
@@ -0,0 +1,13 @@
+import styles from './Button.module.css';
+
+const Button = ({text, type='btn', onClick}) => {
```

Comment:
I dig the way you passed this default prop, it is way less wordy than what I've seen. In the interest of adhering to Next.js/React best practices, we might explore the defaultProps approach.

Not saying either way is better than the other, but I believe I've seen it in the documentation and thought we might explore it to see if there is any added efficiency.